problems/2021-cnrs-benchmark/phase-0/nts.i

```diff
@@ -37,7 +37,6 @@
   vacuum_boundaries = 'bottom left right top'
   create_temperature_var = false
   eigen = true
-  transient = false
```

Comment:
Why is this word removed? Wouldn't it change the behavior? Or is it now deprecated?

Reply:
This change is actually part of #237 which we just merged. This object never required `transient` as an input parameter. But MOOSE tests are now stricter with unused parameters in the input file.